corehq/motech/repeaters/optionvalue.py

```diff
@@ -65,3 +67,8 @@ def __set__(self, obj, value):
 @attr.s
 class OptionSchema:
     options = attr.ib()
```

Comment:
It looks like `OptionSchema` subclasses are objects with `OptionValues`, like a nested version of the objects that `OptionValue` is created for. If that's right, could we use
```python
    options = attr.ib(factory=dict)
```
to make it a little clearer that that's what `options` is for?

It isn't obvious to me why we went with this approach (rather than using a list of keys, or a dataclass, etc.). Can we chat through this over a call?

Reply:
`OptionSchema` is a left-over thing from a false start on attempting to build something like JsonObject's `DocumentSchema` for `OptionValue` (implementing it turned out to be [too complicated](https://github.com/dimagi/commcare-hq/pull/30348#discussion_r784318743)). I think it should be removed since it does not appear to be referenced by production code.

The eventual decision was to use a `DocumentSchema` (JsonObject thing) to wrap/unwrap the value stored in a `OptionValue`-backed attribute on `SQLRepeater` objects where the features of `DocumentSchema` are needed.